### ISSUE TITLE
Sync error ashwini

### ISSUE THREAD

**Issue by mahalakshme** [avniproject/avni-client#1202]
https://avni.freshdesk.com/a/tickets/3266

**Comment by 1t5j0y** [comment]
Issue due to uuid collision for program_encounter.
Long term fix - https://github.com/avniproject/avni-client/issues/1201

Same fix as #1198 
``` 
const conflictingUuid = '3d8861d5-9e40-47d7-91b6-acb553edb2db';
const replaceUuid = '3d8861d5-9e40-47d7-91b6-acb553edb2dc';
```

Awaiting go ahead from user to proceed.

**Comment by 1t5j0y** [comment]
Go ahead received and patch via playground executed and synced.

Notes:
Had to be done with client running as dev connected to Prod by updating SERVER_URL (prod_dev was also throwing permission related errors for the realm file when accessed from Playground).
